cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3001"
+}
```

Comment:
And we're sure this should be the client local host and not localhost:3000? I still have some reservations about this, but if you guys are sure then feel free to resolve this comment and leave this as is

Reply:
client is on 3001.